templates/tease-search.twig

```diff
@@ -58,7 +58,8 @@
 			<p class="search-result-item-content">{{ post.preview()|truncate( 25, true )|raw }}</p>
 
 			{% if ( is_action ) %}
-				<a href="{{ post.link }}" class="btn btn-primary btn-medium btn-small">{{ __( 'Take Action', domain ) }}</a>
+				{%  set link_text = settings['take_action_covers_button_text'] %}
```

Comment:
@kirdia do we need here the default 'Take Action' value in case the setting is not supplied?